Gemfile.lock

```diff
@@ -62,14 +62,14 @@ GEM
       nokogiri (>= 1.6)
     rails-html-sanitizer (1.0.4)
       loofah (~> 2.2, >= 2.2.2)
-    railties (5.2.1)
-      actionpack (= 5.2.1)
-      activesupport (= 5.2.1)
+    railties (5.2.2.1)
+      actionpack (= 5.2.2.1)
+      activesupport (= 5.2.2.1)
       method_source
       rake (>= 0.8.7)
       thor (>= 0.19.0, < 2.0)
-    rake (12.3.1)
-    thor (0.20.0)
+    rake (12.3.2)
```

Comment:
### [rake](https://github.com/ruby/rake)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 12.3.1 → 12.3.2

[_~~change-log~~, [source-code](https://github.com/ruby/rake)_]





<details>
<summary>Commits</summary>

A change of **41** commits. See the full changes on [the compare page](https://github.com/ruby/rake/compare/v12.3.1...v12.3.2).

These are the first 10 commits:
- (acae4a2) [Fix JRuby detection on JRuby 9.2 in cpu_counter.rb](https://github.com/ruby/rake/commit/acae4a2b696d9410c428da735ae6d3364530fd76)
- (c376a93) [Use simpler RUBY_PLATFORM check for java in cpu_counter.rb](https://github.com/ruby/rake/commit/c376a932f93ac0f2dcac66002df71478a315ba42)
- (bdc6406) [Add alias `prereqs`.](https://github.com/ruby/rake/commit/bdc6406a56432a16d65aca9bf6ce82defd9718d2)
- (124a03b) [Merge pull request #268 from take-cheeze/prereqs_alias](https://github.com/ruby/rake/commit/124a03bf4c0db41cd80a41394a9e7c6426e44784)
- (ddceed9) [Merge pull request #265 from jeremyevans/jruby92-fix](https://github.com/ruby/rake/commit/ddceed91a45b1f9dfaf1c3385f0ed4aaf7a53ecf)
- (110cc42) [remove trailing extension name in require](https://github.com/ruby/rake/commit/110cc421bb2b751fb8da24bb050040758bef9db0)
- (f34e2d5) [Fix a typo in lib/rake/application.rb](https://github.com/ruby/rake/commit/f34e2d57f01938eb1b9334b4a9354eab9e585b35)
- (a54a506) [Remove more space after `!` operator](https://github.com/ruby/rake/commit/a54a506b3f82335a258e91708ee66e9c80edc63d)
- (51fdc40) [Merge pull request #276 from felixonmars/patch-1](https://github.com/ruby/rake/commit/51fdc40f4ec38a5ca41ee85e55e2699e78481c03)
- (2a16d03) [Merge pull request #275 from zhustec/master](https://github.com/ruby/rake/commit/2a16d036ccafa51c407a648e081b686ded03e784)


</details>